proms-resources/frontend/src/main/frontend/src/proms/Clinics.jsx

```diff
@@ -29,7 +29,7 @@ import {
   Typography
 } from "@mui/material";
 
-import Fields, { formatString } from "../questionnaireEditor/Fields.jsx";
+import Fields, { camelCaseToWords } from "../questionnaireEditor/LabeledField.jsx";
```

Comment:
`camelCaseToWords` is already imported 5 lines below. And `Fields` can't be imported from `LabeledField.jsx`.
```suggestion
import Fields from "../questionnaireEditor/Fields.jsx";
```